tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "esModuleInterop": true,
+    "exactOptionalPropertyTypes": true,
+    "forceConsistentCasingInFileNames": true,
```

Comment:
Hmm, this seems useful for the module template

Reply:
Issue added here, I'll work on this later today: https://github.com/MetaMask/metamask-module-template/issues/103